osci/preprocess/match_company/company_domain_match_list.yaml

```diff
@@ -461,103 +562,126 @@
     - expediagroup.com
     - hotels.com
     - vrbo.com
+  industry: Travel & Transport
```

Comment:
Sounds good.
Would you mind removing `egencia.com` from the domains as it is now a different company? Otherwise I can do that in a follow-up PR.

Reply:
Thanks for your review. Regarding domain removing, it would be better to do it in a separate PR, so as not to mix changes